make install doesn't exist in makefile?

when trying to follow the install instructions using the makefile I ran into a no target found.

I assumed the error was me using the default make that comes with macs, but after inspecting the Makefile I couldn't find a target called install either.

```shell
➜  ~ git clone https://github.com/mjslabs/ggft.git && cd ggft
make install
Cloning into 'ggft'...
remote: Enumerating objects: 75, done.
remote: Counting objects: 100% (75/75), done.
remote: Compressing objects: 100% (60/60), done.
remote: Total 75 (delta 20), reused 66 (delta 13), pack-reused 0
Unpacking objects: 100% (75/75), 40.15 KiB | 587.00 KiB/s, done.
make: *** No rule to make target `install'.  Stop.
➜  ggft git:(master) make install
```

--- a/Makefile
+++ b/Makefile
@@ -2,9 +2,15 @@ PKG := $(shell head -1 go.mod | sed -e 's/module //')
 PKG_LIST := $(shell go list ${PKG}/...)
 GO_FILES := $(shell find . -name '*.go')
 OUT := $(shell basename ${PKG} | sed -e 's/\.git//')
+DIST := $(OUT).zip
 VERSION := $(shell git describe --tag --long --dirty 2>/dev/null)
 ifeq ($(VERSION),)
 VERSION = development
+endif
+
+INSTALLBIN := $(GOPATH)/bin
+ifeq ($(GOPATH),)
+INSTALLBIN = ~/go/bin
 endif
 
 all: build
@@ -12,19 +18,32 @@ all: build
 .PHONY: build
 build: vet lint ggft
 
+
+ifeq ($(GOOS),windows)
+OUT := $(OUT).exe
+.PHONY: ggft
+endif
 ggft:
-	go build -v -o ${OUT} -ldflags="-X 'github.com/mjslabs/ggft/cmd.version=${VERSION}'" ${PKG}
+	go build -v -o $(OUT) -ldflags="-X 'github.com/mjslabs/ggft/cmd.version=$(VERSION)'" $(PKG)
+
+.PHONY: dist
+dist: ggft
+	zip $(OUT).zip $(OUT)
+
+.PHONY: install
+install: ggft
+	install -D $(OUT) $(INSTALLBIN)
 
 c.out:
-	go test -coverprofile=c.out -v ${PKG}/...
+	go test -coverprofile=c.out -v $(PKG)/...
 
 .PHONY: clean
 clean:
-	rm -f ${OUT} c.out
+	rm -f $(OUT) $(OUT).exe $(OUT).exe.zip $(DIST) c.out
 
 .PHONY: lint
 lint:
-	@for file in ${GO_FILES}; do \
+	@for file in $(GO_FILES); do \
 		golint $$file ; \
 	done
 
@@ -37,4 +56,4 @@ tidy:
 
 .PHONY: vet
 vet:
-	@go vet ${PKG_LIST}
+	@go vet $(PKG_LIST)

